examples/jobs/lsfjob.py

```diff
@@ -53,10 +53,11 @@ def main():
         jd.executable        = '/bin/touch'
         jd.arguments         = ['$FILENAME']
 
-        jd.total_cpu_count   = 16
+        jd.total_cpu_count   = 42
+        jd.total_cpu_count   = 4
 
-        jd.queue             = "regular"
-        jd.project           = "URTG0003" 
+        jd.queue             = "batch"
+        jd.project           = "GEO111" 
```

Comment:
For the future, we should probably use an env variable to store this?

Reply:
We should move to config files.  SAGA supports them now, but they are not yet consistently used in the adaptors...